sdk/schema/plugin.go

```diff
@@ -78,12 +78,6 @@ func (p Plugin) Validate() (bool, ValidationReport) {
 		Severity:    ValidationSeverityError,
 	})
 
-	report.AddCheck(ValidationCheck{
-		Description: "Has no more than one executable defined. Plugins with multiple executables are not supported yet",
-		Assertion:   len(p.Executables) == 1,
-		Severity:    ValidationSeverityError,
-	})
```

Comment:
Let's ensure that we fully add support for multiple executables on the CLI as well, I believe there may be a few instances in the codebase where we may not correctly handle cases where more than one exec is defined per plugin.

Reply:
Following a functional test of the `op plugin` commands, I didn't run into any failures or unexpected behaviour caused by multiple executables. I also looked through the `op` plugin code and couldn't find any areas that require updating, since it was written with the expectation that multiple executables should be supported at some point. But if there's anything in particular you would like me to look into again, please let me know.

One thing though, when running `op plugin list`, or `op plugin init`/`op plugin inspect` without specifying the executable, I noticed that the name defined in `plugin.go` is the one that gets displayed. For instance, since the name defined in `/plugins/aws/plugin.go` is `AWS`, that's what gets displayed for both the `aws` and `cdk` executables, which doesn't seem ideal:

![image](https://user-images.githubusercontent.com/53918631/229648223-77bc2c93-b2d2-45b1-aece-c31d8819562d.png)

We could use `Executable.Name` in these cases instead of `Platform.Name`, but most of the plugin executable names have `CLI` as a suffix and we may not want this. BTW, there seems to be an issue displaying escape sequences for test build plugin names: 

![image](https://user-images.githubusercontent.com/53918631/229648846-889fb0ff-73c2-45d0-acba-7d5c9c85045c.png)

In my opinion, continuing to display the `plugin.go` name as it is right now is probably fine, but I wanted to bring this up in case anyone disagrees.

Reply:
I think the current behaviour of `op plugin list` is the desired one